packages/react-components/lib/robots/robot-info.spec.tsx

```diff
@@ -19,7 +19,17 @@ describe('RobotInfo', () => {
     expect(() => root.getByText('50.00%')).not.toThrow(); // battery
     expect(() => root.getByText('60%')).not.toThrow(); // task progress
     expect(() => root.getByText(/.*underway/)).not.toThrow();
-    expect(() => root.getByText(new Date(0).toLocaleString())).not.toThrow();
+    expect(() =>
+      root.getByText((content, node) => {
+        if (!node) {
+          return false;
+        }
+        const hasText = (node) => node.textContent === new Date(0).toLocaleString();
+        const nodeHasText = hasText(node);
+        const childrenDontHaveText = Array.from(node.children).every((child) => !hasText(child));
+        return nodeHasText && childrenDontHaveText;
```

Comment:
Not too sure if we should be concerned about `childrenDontHaveText`, it seems abit too verbose to test for text, I would expect something like this to be built into testing library if such tests are the intended way to use it.